### ISSUE TITLE
New search bug... again... I know. 

### ISSUE THREAD

**Issue by Thallassa** [Nexus-Mods/web-issues#171]
https://i.imgur.com/s1gPinT.png

Basically I was browsing recent mods and I got real disappointed when there were only 59 results (2 pages of 20 and one page of 19, I counted) instead of the promised 101. 

https://rd.nexusmods.com/skyrim/mods/today/?nav=1&page_size=15

Steps to reproduce:

- look at latest mods on skyrim front page. click load more. 
- click to load 7 days. (It actually stated on sorting by last published, so that's nice). 101 results shown. 
- Browse for 30 minutes or so. Only 59 results actually show up. The remaining pages are empty.

**Comment by mrmason** [comment]
Hey @Thallassa - We don't seem to be able to reproduce this at all - I'm seeing 21 mods today ( 20 on the homepage as it's cached more heavily ). 

We did recently reboot all of the VMS which will have cleared a number of caches which might have refreshed / updated this data.

Are you still seeing the same thing at the moment ?

<img width="349" alt="screenshot 2017-10-23 14 56 39" src="https://user-images.githubusercontent.com/93157/31893053-6b4f3710-b802-11e7-8d98-b848c80f0fe4.png">

**Comment by Thallassa** [comment]
mrmason: The issue isn't on the front page, but if you go click through to "load more", it takes you to the search page. That's where the issue appears as you can see in my imgur link. 

(Note: github refuses to let me upload images directly so please follow my links!)

**Comment by Thallassa** [comment]
Ok after discussing with Ousnius and trying a few things:

It's a tag blocking issue. 

Mods that are not showing up due to blocked tags (whether they're being hidden correctly or not, see #185 ) aren't updating the number. So when I unblocked all tags, I saw all 111 mods correctly. When I had tags blocked, it said there were 111 results but only showed 65ish mods. 

I'm not really sure why there are so many goddamn mods being hidden by these tags, bug #185 is even more serious than I expected, that's half the recent mods not showing up!

When logged out nsfw mods aren't shown - this updates the number correctly (96 mods shown in all places). 

Ousnius was seeing the caching issue as well, but I'm not having trouble with it (probably because Classic updates more slowly and I don't go to SSE often).
